src/configs/index.js

```diff
@@ -1 +1,3 @@
-export const LANGUAGE = 0;
+export const LANGUAGE = 1;
+export const LOGO = 'HORONDI';
+export const URL_LANGUAGE = 'en';
```

Comment:
why do you need URL_LANGUAGE if you don' use it?

Reply:
I use it to create url for category in navbar-left.js